insights/parsers/etc_udev_rules.py

```diff
@@ -52,3 +55,39 @@ class UdevRules40Redhat(LogFileOutput):
     def __init__(self, *args, **kwargs):
         deprecated(UdevRules40Redhat, "Import UdevRules40Redhat from insights.parsers.udev_rules instread.")
         super(UdevRules40Redhat, self).__init__(*args, **kwargs)
+
+
+@parser(Specs.etc_udev_oracle_asm_rules)
+class UdevRulesOracleASM(LogFileOutput):
```

Comment:
@qinpingli based on #2897 could you move the new parser to the udev_rules parser instead of the etc_udev_rules parser?

Reply:
@ryan-blakley Do you mean all the dev rules in `/lib/udev/rules.d` and `/etc/udev/rule.d` dirs should be placed in `udev_rules` based on `All udev rules related parser can be placed in udev_rules module only.`?

Yes, I saw the 40-redhat udev rule was moved to the `udev_rules`, I thought the reason is 40-redhat udev rule is in `/lib/udev/rules.d` dir.

Reply:
@qinpingli yes, since udev rules can be in multiple locations, I think it's more logical to place all udev rule parsers under the udev_rules module instead of splitting them between two modules based on the location.

Reply:
@ryan-blakley got it, thanks! Moved the parser to `udev_rules`, please help take look again, thanks!

Reply:
@qinpingli Awesome thank you, everything looks good to me.